src/NuGet.Clients/NuGet.Tools/NuGetPackage.cs

```diff
@@ -806,11 +806,14 @@ private void BeforeQueryStatusForPowerConsole(object sender, EventArgs args)
                 NuGetUIThreadHelper.JoinableTaskFactory.Run(InitializeMEFAsync);
             }
 
+            var isConsoleBusy = false;
             if (ConsoleStatus != null)
             {
-                var command = (OleMenuCommand)sender;
-                command.Enabled = !ConsoleStatus.Value.IsBusy && !_powerConsoleCommandExecuting;
+                isConsoleBusy = ConsoleStatus.Value.IsBusy;
             }
+
+            var command = (OleMenuCommand)sender;
+            command.Enabled = isConsoleBusy && !_powerConsoleCommandExecuting;
```

Comment:
This is not correct, you reversed the condition. If console is not busy then you want to enable command, not the other way around.

Reply:
yeah, i realized a few mins back. sending out a fix soon.